packages/aws-client/tests/S3/test-multipartCopyObject.js

```diff
@@ -89,6 +90,33 @@ test('multipartCopyObject() copies a file between buckets', async (t) => {
   t.truthy(etag, 'Missing etag in copy response');
 });
 
+test('multipartCopyObject() fails when the maxChunkSize is smaller than the minimum allowed object size', async (t) => {
```

Comment:
should we have a test for a chunksize too large?  or too many parts?

Reply:
chunkSize is 5 MB to 5 GB.  we will need a file > 5GB to test the maximum chunk size.

Reply:
ok.  yeah guess we don't really need that then.